volplugin/handlers.go

```diff
@@ -226,6 +227,7 @@ func mount(master, host string, ttl int) func(http.ResponseWriter, *http.Request
 
 		actualSize, err := volConfig.Options.ActualSize()
 		if err != nil {
+			removeStopChan(uc.Request.Name)
```

Comment:
ah, makes sense, we can leave it as is then. Let me also think about this a bit more and will submit a patch if I find a similar pattern else where.